compiler/rustc_resolve/src/lib.rs

```diff
@@ -961,6 +967,9 @@ pub struct Resolver<'a, 'tcx> {
     /// All non-determined imports.
     indeterminate_imports: Vec<Import<'a>>,
 
+    /// Imports for redundant check.
+    redundant_imports: FxHashMap<NodeId, Import<'a>>,
```

Comment:
```suggestion
    potentially_redundant_imports: FxHashMap<NodeId, Import<'a>>,
```
Could you also put this field next to `potentially_unused_imports`?